Makefile

```diff
@@ -410,3 +408,7 @@ cli:
 
 validate_client:
 	skipper run "python3 ${DEBUG_FLAGS} src/service_client/client_validator.py"
+
+generate_python_client: bring_assisted_service
+	cd assisted-service && skipper make generate-python-client
+	mkdir -p ./.pip && mv assisted-service/build/assisted-installer/assisted-service-client/dist/*.whl ./.pip/
```

Comment:
I would give it more indicative name. Maybe under `build/` or `dist/`

Reply:
`build` is in `.dockerignore`. I can use `dist`, but why is it more fitting ?

Reply:
pip is the package installer in python, if in the future someone will investigate something and see .pip he could think it is related to the package installer.
I prefer to give it a name that someone that reads it will understand immediately